readme.md

```diff
@@ -914,11 +914,11 @@ Assert that `value` is `false`.
 
 ### `.is(value, expected, [message])`
 
-Assert that `value` is equal to `expected` (using `Object.is()` for equality testing).
+Assert that `value` is the same as `expected`. This is based on using [`Object.is()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is).
```

Comment:
The "using" seems superfluous. Either `This is based on` or `This relies on` I think. Also for `.not()`.